src_c/camera_windows.c

```diff
@@ -1,3 +1,25 @@
+/*
+  pygame - Python Game Library
+  Copyright (C) 2000-2001  Pete Shinners
```

Comment:
Sorry, I meant with your copyright in there of course.

Reply:
I think I've got it now, but let me know if I've misinterpreted again.